controller/controller_test.go

```diff
@@ -150,7 +148,7 @@ func newEngineImage(image string, state longhorn.EngineImageState) *longhorn.Eng
 				Version:   "latest",
 				GitCommit: "latest",
 
-				CLIAPIVersion:           4,
+				CLIAPIVersion:           8,
```

Comment:
Why is the `CLIAPIVersion` incremented from `4` to `8` instead of `5`?

Reply:
The CLIAPIVersion was incremented from 4 to 8 in a test case to [match the current version](https://github.com/longhorn/longhorn-engine/blob/55ce0155928121eecbeaa1c24d530a232a68ebe1/pkg/meta/version.go#L5). Are there any concerns?

Reply:
No, just curious. Thank you.